mmv1/products/dns/terraform.yaml

```
@@ -16,6 +16,20 @@ overrides: !ruby/object:Overrides::ResourceOverrides
   ManagedZone: !ruby/object:Overrides::Terraform::ResourceOverride
     id_format: 'projects/{{project}}/managedZones/{{name}}'
     examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "dns_domain_tutorial"
+        primary_resource_id: "default"
+        skip_docs: true
+        vars:
+          dns_compute_instance: "dns-compute-instance"
+          allow_http_traffic: "allow-http-traffic"
+          example_zone_googlecloudexample: "example-zone-googlecloudexample"
+          server_ip_dns_record: "server-ip-dns-record"
+        min_version: beta
+        # ignore_read_extra:
+        #   - "port_range"
+        #   - "target"
+        #   - "ip_address"
```

```suggestion
        # ignore_read_extra:
        #   - "port_range"
        #   - "target"
        #   - "ip_address"
```
These are just copied comments, not needed